index.js

```diff
@@ -47,17 +47,26 @@ NodePreGypGithub.prototype.init = function() {
 		}
 		else throw new Error('A correctly formatted GitHub repository.url was not found within package.json');
 	}
-	
-	hostPrefix = 'https://github.com/' + this.owner + '/' + this.repo + '/releases/download/';
+
+	hostPrefix = 'https://github.com'
+	pathPrefix = '/' + this.owner + '/' + this.repo + '/releases/download/';
 	if(!this.package_json.binary || 'object' !== typeof this.package_json.binary || 'string' !== typeof this.package_json.binary.host){
 		throw new Error('Missing binary.host in package.json');
 	}
 	else if (this.package_json.binary.host.substr(0, hostPrefix.length) !== hostPrefix){
 		throw new Error('binary.host in package.json should begin with: "' + hostPrefix + '"');
 	}
```

Comment:
I was going to ask you to split the above `else if` statement into two, like so:
```javascript
else if (!this.package_json.binary.tag && this.package_json.binary.host.substr(0, hostPrefix.length) !== hostPrefix){
  		throw new Error('binary.host in package.json should begin with: "' + hostPrefix + '" when binary.tag is not used.');
  	}
else if (this.package_json.binary.tag && 'string' === typeof this.package_json.binary.tag && this.package_json.binary.host.substr(0, hostPrefix.length) !== hostPrefix){
  		throw new Error('binary.host in package.json should be equal to: "' + hostPrefix + '"');
  	}
```
but now I am pondering whether it makes sense to try to make this backwards compatible with the old format. What are your thoughts? Regardless, I will be making this a MAJOR version change so I'm not sure there is a point to make it backwards compatible.